packages/ui/src/utils/UiUtil.ts

```diff
@@ -68,5 +68,23 @@ export const UiUtil = {
 
   getWalletName(name: string, short = true) {
     return short ? name.split(' ')[0] : name;
+  },
+
+  formatNumberToLocalString(value: string | number | undefined, decimals = 2) {
+    if (value === undefined) {
+      return '0.00';
+    }
+
+    if (typeof value === 'number') {
+      return value.toLocaleString('en-US', {
```

Comment:
do we always default to the `en-US` locale?

Reply:
im following web decisions so it works similar to them